8/Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV PHP_FPM_MIN_SPARE_SERVERS 1
 ENV PHP_FPM_MAX_SPARE_SERVERS 3
 ENV PHP_FPM_PM_STATUS_PATH /status
 
-ENV XDEBUG_VERSION 3.3.1
+ENV XDEBUG_VERSION 3.2.0
```

Comment:
We can not use this version with `8.3.2` PHP version.

If we want to keep the _old_ version of `xdebug`, we need to remove the `8.3.2` PHP version or create a new and different `Dockerfile`.

Reply:
I added the new `8.3` folder to use the new `xdebug` version only for `8.3.X` php versions.